uaclient/entitlements/tests/test_fips.py

```diff
@@ -39,31 +39,109 @@ def entitlement(fips_entitlement_factory):
 
 
 class TestFIPSEntitlementDefaults:
-    @pytest.mark.parametrize("series", (("xenial"), ("bionic"), ("focal")))
+    @pytest.mark.parametrize(
+        "series, is_container, expected",
+        (
+            (
+                "xenial",
+                False,
+                [
+                    "strongswan",
```

Comment:
I think we can use the constant package variables here

Reply:
Good point!